### ISSUE TITLE
master is broken - revert to 4992dc633797f7d86a0861934513a6bbae2cae95 for working version

### ISSUE THREAD

**Issue by fisch0920** [docstrap/docstrap#116]
After some digging, I've found that the last stable / working commit was [4992dc633797f7d86a0861934513a6bbae2cae95](https://github.com/terryweiss/docstrap/commit/4992dc633797f7d86a0861934513a6bbae2cae95).  The subsequent commit [95dadc4fd634f42aff3d5211738ec49db1c81f70](https://github.com/terryweiss/docstrap/commit/95dadc4fd634f42aff3d5211738ec49db1c81f70) and all commits thereafter break the build.

This includes, but may not be limited to:
* removing local moment.js and replacing with require("moment")
* malformed templates leading to underscore template errors
* duplicate header / titles everywhere

A fixed, working fork is available [here](https://github.com/fisch0920/docstrap).

There were a lot of useful additions haphazardly piled into the merges on May 15, and we should cherry pick some of them going forwards, but for anyone else looking for a stable version of docstrap, start at 4992dc633797f7d86a0861934513a6bbae2cae95.

**Comment by Riobe** [comment]
In package.json you can include ink-docstrap like the following:

```json
"devDependencies": {
    "ink-docstrap": "git://github.com/terryweiss/docstrap.git#4992dc633797f7d86a0861934513a6bbae2cae95",
}
```

This will also fix it while still relying on the main repo. Thanks to @fisch0920 for finding the last stable version.

**Comment by lukeapage** [comment]
@terryweiss any chance you can give permission to someone to help sort this mess out? its so frustrating! @fisch0920 if @terryweiss if non communicative and you have the time, you can ask npm to give you acess to docstrap on npm, then you could publish from your repo and we could all move to that.

**Comment by fisch0920** [comment]
@terryweiss I'm happy to help out or take over as a maintainer.  Another option would be just to add me as a collaborator on this repo so I have merge access.  This is probably preferable because a lot of people would still end up here even if we moved npm to another fork.

**Comment by lukeapage** [comment]
It would be best to create a docstrap organisation actually. I'm sure once
npm points at a new place we can get links updated on any active repo
pointing at this.

How about I create the organisation,  add people as admins and then contact
npm - im sure terry has a filter on emails and maybe that will prompt him.
if the repo is moved rather than forked then it will redirect.

**Comment by fisch0920** [comment]
@lukeapage sounds like a good plan.

**Comment by lukeapage** [comment]
done and released as 0.5.3
